tests/test_admin_async.py

```diff
@@ -463,6 +533,11 @@ async def test_update_get_page(client: AsyncClient) -> None:
         == 1
     )
     assert response.text.count('<option selected value="1">Address 1</option>') == 1
+    assert (
+        response.text.count('<select class="form-control" id="profile" name="profile">')
```

Comment:
Sorry I think I wasn't clear in my previous comment, I think we should get rid of adding tests in both `test_admin_sync` and `test_admin_async` unless we have a very good reason.
For example this is checking the `select` which is already tested by OneToMany relation. I think we can avoid both these files changes.

Reply:
Thanks for the follow-up. I'm sorry but I can't judge which tests are really unnecessary because I don't fully understand your implementations. I think there are many potential pitfalls between `OneToOne` (single object) and `OneToMany` (array of objects), so I basically added the checks same as for `Address`.

For example, the above test checks that the response has a `<select>` tag *without* `multiple` attribute. IMO, these tests are verbose but it seems to be useful to detect some kind of degradation in the future.

So, I want you to merge it as it is, and remove ones you think unnecessary in another patch (or remove all tests to add them later?). What do you think?

Reply:
Yeah that's ok, I can clean it up which I will need to do anyway, let's then focus to make this working and merge it as is.

Reply:
Thanks!